### ISSUE TITLE
error

### ISSUE THREAD

**Issue by krgy12138** [tintin327/predrnn_pp_Pytorch#2]
when use loss = self.loss_func(gen_images,images[:,1:]) in model.py, the input should be between 0 and 1.But gen_images is out of range
